app/workers/terms_of_use/sign_up_service_updater_job.rb

```diff
@@ -5,6 +5,13 @@
 module TermsOfUse
   class SignUpServiceUpdaterJob
     include Sidekiq::Worker
+    sidekiq_options retry: 15 # 2.1 days using exponential backoff
+
+    sidekiq_retries_exhausted do |job, _ex|
+      Rails.logger.warn(
+        "Retries exhausted for #{job['name']} with args #{job['args']}: #{job['error_message']}}"
```

Comment:
Im not sure this is possible, do we record the response from the SuS API as to why the try failed? Is that what will be in `error_message`?

Reply:
The error message will either be this bad boy right [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/mobile_application_platform/sign_up/service.rb#L31) for acceptance errors, or this [one](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/mobile_application_platform/sign_up/service.rb#L41) for decline errors.

Reply:
Note that the error message will be whatever message sent by ANY exception raised, so it could be more than just those two cases that we see error messages for.